test/arrays/bradc/enumarray3.chpl

```diff
@@ -0,0 +1,19 @@
+enum classVals { S, W, A, B, C, D, O };
+
+var DClass = {classVals.S..classVals.O};
```

Comment:
Will the resulting domain be a ``domain(1)``?

What if the enum had specified values with gaps, or out of order values?

Will it be possible to write this as ``var DClass = {classVals};``?

Reply:
A `domain(rank=1, idxType=classVals)`, yes.  The current notion is that ranges over enums would hit all of the enums in the order of the enumeration independent of underlying numerical value.  Of course, the compiler would have optimization opportunities when the underlying numerical values happened to be consecutive as in this case.

Reply:
> Will it be possible to write this as ``var DClass = {classVals};``?

I'd guess "no" though this might be considered a reasonable shorthand for `domain(classVals)` (an associative domain of enums).